__tests__/app.test.js

```diff
@@ -115,3 +117,51 @@ describe("Error Handling 404 ", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("return 200 status, should get a comment by its article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(Array.isArray(comments)).toBe(true);
+        expect(comments.length).toBeGreaterThan(0)
```

Comment:
check the exactly length

Reply:
Please amend this line of code as per Alice's previous comment.